src/UI/Factory.php

```diff
@@ -336,4 +337,69 @@ public function panel();
 	 **/
 	public function modal();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Popovers can be used if space is scarce to display secondary information such as a preview of
+	 *     some item or to offer limited user interaction.
+	 *   composition: >
+	 *     Popovers consist of a layer displayed above all other content.
+	 *     The content of the Popover depends on the functionality it performs.
+	 *     A Popover MAY display a title above its content.
+	 *     All Popovers contain a pointer pointing from the Popover to the Trigger of the Popover.
+	 *   effect: >
+	 *     Popovers are shown by clicking a Triggerer component such as a Button or Glyph.
+	 *     The position of the Popover is calculated automatically be default. However, it is possible to
+	 *     specify if the popover appears horizontal (left, right) or vertical (top, bottom) relative to
+	 *     its Triggerer component. Popovers disappear by clicking anywhere outside the Popover or by pressing
+	 *     the ESC key.
+	 * rivals: >
+	 *   Modals: >
+	 *     Modals hide all other content while Popovers do not prevent interaction with other parts
+	 *     of the current context.
+	 * context:
+	 * rules:
+	 *   usage:
+	 *     1: >
+	 *        Popovers MUST NOT contain horizontal scrollbars. The renderer of the content component is responsible
+	 *        that the content does not exceed the max width from the Popover.
+	 *     2: >
+	 *        Popovers MAY contain vertical scrollbars. The renderer of the content component is responsible to
+	 *        set the max height and showing the vertical scrollbars.
```

Comment:
I think here, the second part might be actually useful for the consumer. Maybe just rephrase a little to avoid the word renderer. E.g. "The content component is responsible to set it's own height and show it's vertical scrollbars."